### PR TITLE
Extend test suite information in the README

### PR DESCRIPTION
**Pull request by nodeg**
This will add some of the frequent information when talking about the test suite like on what hypervisor it runs and if it uses a mirror.

### CODE REVIEW

**Review by elariekerboull — APPROVED** [review]
Great idea, thanks for that!

File: README.md [hunk 1]
```diff
@@ -53,7 +53,9 @@ the build is reproducible.
 image for testing Pull Requests built with the open build service. This needs to be copied manually.
 - http://minima-mirror.mgr.prv.suse.net/distribution/leap/: oss and non-oss repositories.
 
-## Used image versions in the CI test suite
+## CI Test suite
```
Comment (vandabarata) [inline README.md:56]:
```suggestion
## CI Testsuite
```

**Review by vandabarata — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nodeg) [reply README.md:56]: Nope. We use `test suite` for quite some time now 😉

**Review by srbarrios — APPROVED** [review]

**Review by Bischoff — COMMENTED** [review]
Ah, that is a recurring problem.

We said several times that this information changes too often to be put in some documentation file.
The current theory is to look in grafana - however that is not convenient.

In short, I like this PR, but it goes against what we said previously. That's not to say we cannot change our mind.

Small remark: we could add information for BV (but then have to change the title).

**Review by srbarrios — APPROVED** [review]
I would say it's enough maintaining this info in the infrastructure repository and also in our grafana dashboards. This will cause some extra maintenance. But I would not oppose to merge it, just giving my personal opinion about that kind of doc.

**Comment by nodeg** [comment]
I am also not a big fan of having duplicate information but sometimes I need information and I do not know where to find it. We could also of course extend our Confluence QE landing page with all the information or links to the information to have everything in in one place. So we instantly know where to look.

**Comment by nodeg** [comment]
Ok. I guess I am closing this PR in favor of adding/linking the information in our Confluence space. Anyway thank you for your reviews and comments.

**Pull request closed without merging**